README.md

```diff
@@ -34,16 +39,22 @@ Checks if the input value is a valid semver version.
 #### Return
 - **Boolean** Returns `true` if the input is a valid semver version or `false` otherwise.
 
-## How to contribute
+
+
+## :yum: How to contribute
 Have an idea? Found a bug? See [how to contribute][contributing].
 
-## Where is this library used?
+## :dizzy: Where is this library used?
 If you are using this library in one of your projects, add it in this list. :sparkles:
 
-- [kanpai](https://github.com/egoist/kanpai) - An alternative to `npm publish`
-- [nwjs](https://github.com/egoist/nwjs) - NW.js version manager
 
-## License
+ - [`gpm`](https://github.com/IonicaBizau/gpm)—npm + git = gpm - Install NPM packages and dependencies from git repositories.
+ - [`kanpai`](https://github.com/egoist/kanpai#readme) (by EGOIST)—🍻  Kanpai to publish a new version of your module!
```

Comment:
@egoist I'm updating the docs in all my repositories, whitelisting the npm files too. :rocket: Just look at my recent public activity. :smile: